hudi-client/src/test/java/org/apache/hudi/testutils/Assertions.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.testutils;
+
+import org.apache.hudi.client.WriteStatus;
+
+import org.apache.spark.api.java.JavaRDD;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+/**
+ * Commonly used assertion functions.
+ */
+public class Assertions {
+
+  /**
+   * Assert no failures in writing hoodie files.
+   */
+  public static void assertNoWriteErrors(JavaRDD<WriteStatus> statuses) {
```

Comment:
Hi @xushiyan I have a concern about open this method. Currently, @wangxianghu is refactoring client module #1727 . He is trying to decouple the dependency with Spark APIs. Maybe we'd better have this interface:

```
  public static void assertNoWriteErrors(List<WriteStatus> statuses) {
    //...
  }
```

WDYT?

Reply:
Ok that method is also there. just need to remove the overloading version for RDD. changing it now.

Reply:
Updated. Thanks @yanghua

Reply:
Thanks for addressing my concern.